pkgs/applications/graphics/oculante/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, pkg-config
+, openssl
+, fontconfig
+, nasm
+, netcat
+, libX11
+, libXcursor
+, libXrandr
+, libXi
+, libGL
+, stdenv
+, libobjc
```

Comment:
this is not a top-level attribute, use `darwin` instead

Reply:
I am not sure about this since inheriting darwin seems pretty common in nixpkgs - https://github.com/NixOS/nixpkgs/blob/59004d7cf0701f1c1f12d1a91cfe78109ded1ed0/pkgs/development/web/deno/default.nix#L9 ?

Reply:
https://github.com/NixOS/nixpkgs/issues/204303